src/modules/notifications.tsx

```diff
@@ -17,14 +30,14 @@ export function showNotification(err) {
   }
 
   Store.addNotification({
-    content: <Toast errorMsg={message} status={status} />,
+    content: <Toast errorMsg={message} status={status} jobId={jobId} />,
     insert: 'top',
     container: 'top-right',
     animationIn: ['animate__animated', 'animate__slideInRight'],
     animationOut: ['animate__animated', 'animate__slideOutRight'],
     dismiss: {
       duration: 5000,
-      touch: false,
+      touch: true,
```

Comment:
Want to confirm the expected behavior here: I'm seeing that all toast notifications now must be clicked on to close, including those without associated jobIds.

I think I like this better, we've heard from users that their toast disappeared before they could save the information.  But the behavior is not 100% clear.  Would it be possible to give toasts an "x" icon that you click to close?  That may be OOS.

Reply:
Yeah, I was trying to style it and it was driving me nuts so I just left it on.  I think it's better this way.  I can certainly style...I want to sync with Lou about the job link anyways

Reply:
This is an interesting change - I could imagine a user trying to copy/paste the error message, but that would actually dismiss the message, right? I think an "x" icon would be a good idea.